### PR TITLE
bescort.lic: Add support for the mriss/lang balloon.

### PR DESCRIPTION
**Pull request by Sarvatt**
Yes baloon is misspelled on purpose internally, keeping in tradition with escort.lic. :)

Closes: #2554

### CODE REVIEW

**Comment by Sarvatt** [comment]
i'm open to making it balloon too. just find this funny since escort.lic never worked with it and keeping history :)

**Comment by rpherbig** [comment]
I'd much prefer 'balloon'.

**Comment by Sarvatt** [comment]
fixed :)

File: bescort.lic [hunk 1]
```diff
@@ -658,6 +664,36 @@ class Bescort
...
+  def take_balloon(mode)
+    case mode
+    when 'mriss'
+      manual_go2(8793)
+      if wealth('Therenborough') < 10_000
+        echo('Get money you slob!')
+        return
+      end
+    when 'langenfirth'
+      if wealth('Mer'Kresh') < 10_000
```
Comment (rpherbig) [inline bescort.lic]: This needs to either be double-quotes or escape the `'`... :sad_trombone:

**Pull request merged**